examples/basic/android/settings.gradle

```diff
@@ -1,12 +1,12 @@
 rootProject.name = 'VideoPlayer'
 
-include ':app',
-        ':react-native-video',
-        ':react-native-video-exoplayer'
+include ':react-native-video'
+project(':react-native-video').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-video/android-exoplayer')
 
 // Quick Local Development
 //project(':react-native-video').projectDir = new File(rootProject.projectDir, '../../android')
 //project(':react-native-video-exoplayer').projectDir = new File(rootProject.projectDir, '../../android-exoplayer')
 
-project(':react-native-video').projectDir = new File(rootProject.projectDir, '../../android')
-project(':react-native-video-exoplayer').projectDir = new File(rootProject.projectDir, '../../android-exoplayer')
+// project(':react-native-video').projectDir = new File(rootProject.projectDir, '../../android')
+// project(':react-native-video-exoplayer').projectDir = new File(rootProject.projectDir, '../../android-exoplayer')
```

Comment:
these 2 lines are repeated above

Reply:
i deleted and commented duplicated line in the current version and react-native link again the project

Reply:
I can see that, but please just remove it, even if this is only example code. People reference examples and code like this accounts for more confusion than necessary.

Reply:
yes, thank you for your report

Reply:
i have just deleted repeated code, and made a commit